lib/matplotlib/backends/web_backend/nbagg_mpl.js

```diff
@@ -123,6 +123,22 @@ mpl.figure.prototype._init_toolbar = function() {
     titlebar.prepend(buttongrp);
 }
 
+
+mpl.figure.prototype._canvas_extra_style = function(e){
+    // this is important to make the div 'focusable
+    e.attr('tabindex', 0)
+    // reach out to IPython and tell the keyboard manager to turn it's self
+    // off when our div gets focus
+
+    // location in version 3
+    // IPython.notebook.keyboard_manager.register_events(canvas_div);
```

Comment:
FWIW, they kept `IPython.keyboard_manager` as an alias for backward compatibility.